extensions/lifecycle/tasks/LifecycleUpdateExpirationTask.js

```diff
@@ -126,7 +126,8 @@ class LifecycleUpdateExpirationTask extends BackbeatTask {
               })
               .setAttribute('source', entry.getAttribute('source'))
               .setAttribute('serviceName', 'lifecycle-transition')
-              .setAttribute('target.locations', locations);
+              .setAttribute('target.locations', locations)
+              .setAttribute('target.accountId', accountId);
```

Comment:
should also (try to) pass the `ownerId`, for good measure (like in the other cases)